gulp/tasks/images.js

```diff
@@ -0,0 +1,34 @@
+import imagemin from "gulp-imagemin";
+
+export function prepareImages() {
+	return app.gulp.src(app.path.src.images)
+		.pipe(app.plugins.plumber(
+			app.plugins.notify.onError({
+				title: "prepareImages",
+				message: "Error: <%= error.message %>"
+			})
+		))
+		.pipe(
+			app.plugins.if(
+				app.isBuild,
+				app.plugins.newer(app.path.build.images)
```

Comment:
as I see you import plugins (such as imagemin) in this file which is used only for specific tasks and you use app.plugins... for plugins which are used all over the gulp tasks.

Also I see that `newer` is used only in images, so the question is - why do you use it like global one?

Reply:
Done

Reply:
Done